src/data/nat/gcd.lean

```diff
@@ -191,6 +191,10 @@ or.elim (eq_zero_or_pos k)
     by rw [gcd_mul_lcm, ←gcd_mul_right, mul_comm n k];
        exact dvd_gcd (mul_dvd_mul_left _ H2) (mul_dvd_mul_right H1 _))
 
+lemma lcm_dvd_iff {m n k : ℕ} : lcm m n ∣ k ↔ m ∣ k ∧ n ∣ k :=
+⟨λ h, ⟨dvd_trans (dvd_lcm_left _ _) h, dvd_trans (dvd_lcm_right _ _) h⟩,
+  and_imp.2 lcm_dvd⟩
+
```

Comment:
Surprised we didn't have this result yet, to me it is the specification of `lcm`!

Reply:
We have this for the generalization, gcd_monoid

Reply:
Aha! It looks like importing `gcd_monoid` into this file would rearrange the import structure enough that it's better to just keep it as-is.